core/src/trezor/lvglui/scrs/components/overlay.py

```
@@ -0,0 +1,45 @@
+        self.process_bar.align(lv.ALIGN.CENTER, 0, 0)
+        self.process_bar.set_range(0, max_value)
+        self.process_bar.set_value(value, lv.ANIM.OFF)
+        self.precess_value = lv.label(self)
+        self.precess_value.set_text(f"{value}/{max_value}")
+        self.precess_value.add_style(
+            StyleWrapper().text_font(font_GeistRegular26).text_color(lv_colors.WHITE),
+            0,
+        )
+        self.precess_value.align_to(self.process_bar, lv.ALIGN.OUT_TOP_MID, 0, -16)
+
+    def set_value(self, value: int):
```

Ruff (0.8.2)

Missing return type annotation for public function `set_value`

Add return type annotation: `None`

(ANN201)